docs/development/testguide.rst

```diff
@@ -38,76 +45,43 @@ separately.
 setup.py test
 -------------
 
-The safest way to run the astropy test suite is via the setup command ``test``.
-This is invoked by running ``python setup.py test`` while in the astropy source
-code directory. Run ``python setup.py test --help`` to see the options to the
+Astropy and the affiliated package template provide a ``test`` setup command,
+invoked by running ``python setup.py test`` while in the package root
+directory. Run ``python setup.py test --help`` to see the options to the
 test command.
 
-Turn on PEP8 checking by passing ``--pep8`` to the ``test`` command. This will
-turn off regular testing and enable PEP8 testing.
+Since ``python setup.py test`` wraps the widely-used pytest framework, you may
+from time to time want to pass options to the ``pytest`` command itself. For
+example, the ``-x`` option to stop after the first failure can be passed
+through with the ``--args`` argument::
 
-.. note::
+    > python setup.py test --args "-x"
```

Comment:
Pretty sure it is `--args="-x"` (with equal sign), as that is what I use in my own package. Also, you can pass in multiple options using `--args='-x --y="xyz"'`, if you want to make the document even more detailed.